src/compas_blender/artists/pointartist.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Union
+
+import bpy
+
+import compas_blender
+from compas.artists import PrimitiveArtist
+from compas.geometry import Point
+from compas.utilities import RGBColor
+from compas_blender.artists import BlenderArtist
+
+
+__all__ = ["PointArtist"]
```

Comment:
i started to remove all of this since we don't do any dynamic importing anymore...

Reply:
will do